metadata/src/test/java/org/apache/kafka/controller/QuorumControllerTest.java

```diff
@@ -717,7 +717,7 @@ public void testUnregisterBroker() throws Throwable {
                     setBrokerId(0).
                     setClusterId(active.clusterId()).
                     setIncarnationId(Uuid.fromString("kxAT73dKQsitIedpiPtwBA")).
-                    setFeatures(brokerFeatures(MetadataVersion.IBP_3_0_IV1, MetadataVersion.IBP_3_8_IV0)).
+                    setFeatures(brokerFeatures(MetadataVersion.IBP_3_0_IV1, MetadataVersion.IBP_3_8_IV1)).
```

Comment:
Should we change IBP_3_8_IV0 to IBP_3_8_IV1 in `testUncleanShutdownBroker()`? It is testing ELR, but we want to make sure ELR works under the latest version too, right?

Reply:
Hopefully this ought to be addressed in the next commit as well